repository/sparql_reader.go

```diff
@@ -39,20 +40,17 @@ func (r *SparqlReader) queryPage(sparqlForOffset func(offset int) string, f func
 	for {
 		offset := page * tripleLimit()
 		query := sparqlForOffset(offset)
-		log.Printf("[SPARQL] %s", query)
 		results, err := r.Repo.Query(query)
-		log.Printf("Returned from query")
 		if err != nil {
-			log.Printf("Returning error: %s", err)
 			return err
 		}
-		log.Printf("Counting results")
 		resultCount := len(results.Solutions())
-		log.Printf("[RESULTS] %v", resultCount)
+		log.Printf("[SPARQL RESULTS] %v results for page %v", resultCount, offset)
 		if resultCount == 0 {
 			break
 		}
-		if err = f(results); err != nil {
+		err = f(results)
```

Comment:
Hmm, what's this change for?

Reply:
I don't trust it.

OK, that's not fair. I'm trying to make sure that all of the errors are passed up the stack correctly and trying to remove any point of failure. Which is not a very compelling reason.

Reply:
OK. The `if condition assignment` is idiomatic Go, but I don't care deeply about this change. The PR is approved either way; up to you if you want to change then merge or just merge. :smile: